docker-compose.yml

```diff
@@ -46,11 +46,29 @@ services:
             - agenta-network
         extra_hosts:
             - "host.docker.internal:host-gateway"
+        command: "uvicorn agenta_backend.main:app --host 0.0.0.0 --port 8000 --reload --log-level info --root-path /api"
         depends_on:
             postgres:
                 condition: service_healthy
+            migrate:
+                condition: service_completed_successfully
         restart: always
 
+    migrate:
```

Comment:
Referring to chatgpt we don't need to run this each time we start the backend. 
I also don't understand having this in a separate container where the image is the same used in the backend.
https://chatgpt.com/share/62c4d950-a55f-41e8-90f9-708f44065bf9

Reply:
Got it.